### DISCUSSION TITLE
Feature: Add GitHub Alert Support to README.md Markdown

### DISCUSSION THREAD

**Discussion by babakfp** [community/community#143680] [npm] [answered]
How it looks like on NPM:

![image](https://github.com/user-attachments/assets/e0e8ea77-bcbb-442f-877f-4e1430aef2e6)

on GitHub:

![image](https://github.com/user-attachments/assets/84df466a-f7d3-470e-8db0-1ead505a8bd0)

**Comment by monishcm** [comment]
Please can you elaborate? Is the UX you are referring to?

**Reply by babakfp** [reply]
https://github.com/orgs/community/discussions/16925

GitHub supports this Markdown syntax. When we use this syntax in `README.md` file, it looks like the first screenshot on NPM, but it should look like the second screenshot (the same as how it is on GitHub). Perhaps a light mode version would be better because NPM doesn't have dark mode.

> [!NOTE]
> This isn't just for the `README.md` file, but for all Markdown files.

**Reply by monishcm** [reply, marked as answer]
This is not something that we are looking to implement on a short term. Thanks for your feedback and I'll add this to our backlog to be addressed overtime.
